code/style.css

```diff
@@ -2,6 +2,195 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
+*{
+  margin: 0;
+  padding: 0;
+  border: 0 none;
+}
+
+body, html {
+  height: 100%;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+height: 80%;
+width: auto;
+display:flex;
+align-items:center;
+justify-content: center;
+text-align: center;
+background-image:
+url("https://images.unsplash.com/photo-1474440692490-2e83ae13ba29?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=2850&q=80");
+background-size:cover;
+background-repeat: no-repeat;
+background-position: center;
+position: relative;
+}
+
+h1 {
+  color: whitesmoke;
+}
+
+h2 {
+  color: whitesmoke;
+}
+
+h3 {
+  font-size: 30px;
+}
+
+h4 {
+font-size: 15px;
+}
+
+
+.caption {
+  text-align:center;
+  color:#ffffff;
+}
+
+.caption h1 {
+
+  text-transform: uppercase;
+  font-size:60px;
+  font-family: 'Rampart One', cursive;
+  margin-bottom:0.5rem;
+}
+
+.caption h2 {
+  font-weight:400;
+  font-size:40px;
+  margin:0;
+} 
+
+.info-text {
+  text-align: center;
+  background-color: #7c94ae;
+  padding: 80px;
+}
+
+input {
+  width: 100%;
+}
+
+input[type=text] {
+  width: 100%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  display: inline-block; 
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+input[type=password] {
+  width: 100%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  display: inline-block; 
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+input[type=submit] {
+  width: 100%;
+  background-color: #88999a;
+  color: white;
+  padding: 14px 20px;
+  margin: 8px 0;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+}
+
+input[type=submit]:hover {
+  background-color: #b0c098;
+}
+
+.input-header {
+  padding-bottom: 10px;
+  margin-right: 10px;
+}
+
+ .form-questions {
+  display: flex;
+  flex-wrap: wrap;
+  width: auto;
+  margin-top: 10px;
+}
+
+.headline {
+  padding-bottom: 20px;
+}
+
+.content-box {
```

Comment:
I think for the next time it would be great to add a plus sign to make it clear that there is something that can pop up.